main_test.go

```diff
@@ -15,22 +15,37 @@ import (
 // unit tests of individual functions
 var _ = Describe("CfDownload", func() {
 	var args []string
+	var paths []string
 
-	BeforeEach(func() {
-		args = make([]string, 7)
-	})
-
-	Describe("Test Flag functionality", func() {
+	Describe("Test ParseArgs functionality", func() {
 
 		Context("Check if overWrite flag works", func() {
 			It("Should set the overwrite_flag", func() {
+				args = make([]string, 4)
```

Comment:
Maybe you can't because `args` is defined earlier, but uninitialized? I could maybe see it happening because we didn't do a `:=`-type assignment.